src/main/java/frc/robot/Robot.java

```diff
@@ -28,6 +28,7 @@ public void robotPeriodic() {
 
   @Override
   public void disabledInit() {
+    robotContainer.shooterHoodSubsystem.setPercentage(0);
```

Comment:
Just a suggestion, but you could handle this in the subsystem using a [Trigger](https://docs.wpilib.org/en/latest/docs/software/commandbased/binding-commands-to-triggers.html). I did in [LimelightSubsystem](https://github.com/STMARobotics/frc-7028-2020/blob/master/src/main/java/frc/robot/subsystems/LimelightSubsystem.java#L52) to enable/disable the green lights. It would be something like this:
```
new Trigger(() -> RobotState.isDisabled()).whenActive(setPercentage(0));
```